eldritch/test_usables.py

```diff
@@ -419,5 +516,43 @@ def testMultipleOptions(self):
     self.assertTrue(self.char.possessions[0].exhausted)
 
 
+class MistsTest(EventTest):
+  def setUp(self):
+    super().setUp()
+    self.char.possessions.append(items.Mists(0))
+
+  def testCombatEvade(self):
+    monster = monsters.Cultist()
+    self.state.event_stack.append(EvadeOrCombat(self.char, monster))
+    evade_choice = self.resolve_to_choice(MultipleChoice)
+    evade_choice.resolve(self.state, "Evade")
+    self.resolve_to_usable(0, "Mists0", events.CastSpell)
+    self.state.event_stack.append(self.state.usables[0]["Mists0"])
+    choice = self.resolve_to_choice(MultipleChoice)
+    choice.resolve(self.state, "Cast")
+    with mock.patch.object(events.random, "randint", new=mock.MagicMock(return_value=5)):
+      self.resolve_until_done()
+
+  def testCastAfterFail(self):
+    monster = monsters.Cultist()
+    self.char.clues = 1
+    self.state.event_stack.append(EvadeOrCombat(self.char, monster))
+    evade_choice = self.resolve_to_choice(MultipleChoice)
+    evade_choice.resolve(self.state, "Evade")
+    self.resolve_to_usable(0, "Mists0", events.CastSpell)
+    self.state.done_using[0] = True
+    with mock.patch.object(events.random, "randint", new=mock.MagicMock(return_value=4)):
+      self.resolve_to_usable(0, "Mists0", events.CastSpell)
+    self.state.event_stack.append(self.state.usables[0]["Mists0"])
+    choice = self.resolve_to_choice(MultipleChoice)
+    choice.resolve(self.state, "Cast")
+    with mock.patch.object(events.random, "randint", new=mock.MagicMock(return_value=5)):
+      spend_choice = self.resolve_to_choice(SpendChoice)
+    spend_choice.resolve(self.state, "Done")
+    self.resolve_until_done()
+
```

Comment:
Same comment as above - recommend at least one test where they fail to cast, and you validate that it's exhausted and no longer usable.